opensearch/services/datasets/datasets.php

```diff
@@ -162,6 +178,19 @@ public function getSearchResultOutput(array $result) {
 			$html .= "</span>";
 		}
 
+		//data types
```

Comment:
This is adding a datatype value to the result set for datasets.  Is that part of this feature as well?